README.md

```diff
@@ -1 +1,162 @@
-# roadmap
\ No newline at end of file
+## Kabanero Foundation in a Kubernetes Cluster Prerequisites 
+
+### Software Installation
+
+- [OKD](https://www.okd.io/) v3.11.0+
+- [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager/releases) 0.10.0+
+
+
+### Cluster Hardware Capacity
+
+The full suite of Kabanero foundation components include: 
+  - Che
```

Comment:
Do we have Che yet?

Reply:
I asked about the release of the RC2 Che Operator but haven't heard back from anyone. Also not sure when they plan to put the version in the kabanero repo as I haven't heard anything around that.

Reply:
will remove until its delivered